scripts/build.py

```diff
@@ -47,79 +56,79 @@
 PrintedCompilingImages = False #Used to tell the script whether or not the string "Compiling Images" has been printed
 
 def run_command(cmd):
-	try:
```

Comment:
it's very messy that the file used both space/tab indent and that's why I corrected it.